indico/modules/events/editing/controllers/backend/management.py

```diff
@@ -78,5 +78,12 @@ def _process_PATCH(self, data):
     def _process_DELETE(self):
         if EditingRevisionFile.query.with_parent(self.file_type).has_rows():
             raise ExpectedError(_('Cannot delete file type which already has files'))
+        if self.file_type.publishable:
+            types = (EditingFileType.query
```

Comment:
I'd name this `is_last` instead of `types`, since the latter implies more a list of types instead of a bool